.github/workflows/dev.yaml

```diff
@@ -21,24 +33,23 @@ jobs:
       run: npm ci
     - name: Build
       run: npm run test:functional:ci:build
-    - name: 'BrowserStack Env Setup'
-      uses: 'browserstack/github-actions/setup-env@master'
-      with:
-        username:  ${{ secrets.BROWSERSTACK_USERNAME }}
-        access-key: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
-    - name: 'BrowserStackLocal Setup'
-      uses: 'browserstack/github-actions/setup-local@master'
-      with:
-        local-testing: start
-        local-identifier: random
-    - name: Run Unit & Functional tests on BS
+    - name: Unit Test
+      run: npm run test:unit:browserstack:ci
+    - name: Functional e2e Tests
+      run: |
+        npx testcafe ${{ matrix.browser }} test/functional/specs/Audiences/*.js,
```

Comment:
The only danger I see her is if we add more tests, we need to remember to update this file. Any way around this?

Reply:
@shammowla and I discussed this. I've recommended that we create a script that finds all spec files and then runs them in, say, batches of 10. After you left the release party, we decided to write that script later (@shammowla, please create a ticket for yourself so you don't forget).

Reply:
thanks @Aaronius